Zara Antonia-Maria/L4/3.13/main.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    float v[1001],value;
+    int n,i;
+    printf("Input the number of elements in the sequence: ");
+    scanf("%d",&n);
+    printf("\nInput the elements: ");
+    for(i=1;i<=n;i++)
+    {
+        scanf("%f",&v[i]);
+    }
+    printf("\nInput the value that is to be searched: ");
+    scanf("%f",&value);
+    i=1;
+    while(v[i]<value)
```

Comment:
This is exactly what the problem asked of you, you did it right!
And also, elegant solution too